plugins/CoreAdminHome/Commands/InvalidateReportData.php

```diff
@@ -106,6 +106,22 @@ protected function execute(InputInterface $input, OutputInterface $output)
                         }
                         $logger->info($message);
                     } else {
+                        /**
+                         * Trigger the activity that tracks when a customer requests invalidation.
+                         * Having the post here results in multiple activity logs if there are multiple segments and/or
+                         * periods, but all of the dates and other fields are pretty much in the right format.
+                         */
+                        Piwik::postEvent('InvalidateReports.invalidateReports', [[
```

Comment:
Good eye. It definitely looks that way, but I'm matching the format that the API events use, since the activity processes an API event as well. I could have updated the activity to be smart enough to take both formats, but figured it was easier just to nest the array.